addon/components/sortable-group.js

```diff
@@ -116,6 +116,13 @@ export default Component.extend({
     let items = this.get('sortedItems');
     let groupModel = this.get('model');
     let itemModels = items.mapBy('model');
+    let draggedItem = items.filter(i => get(i, 'wasDropped'))[0];
```

Comment:
i'd use `find` here instead of `filter` and `[0]`, but didn't want to change @scottmessinger's initial PR except to make tests pass.